integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/SparkCarbonDataSourceTest.scala

```diff
@@ -1779,7 +1858,7 @@ class SparkCarbonDataSourceTest extends FunSuite with BeforeAndAfterAll {
     assert(ex.getMessage.contains("column: abc specified in inverted index columns does not exist in schema"))
   }
 
-  var writerPath = new File(this.getClass.getResource("/").getPath
+  var WriterOutputPath = new File(this.getClass.getResource("/").getPath
```

Comment:
why to change the variable name? 
usually variable names should start with smallcase. Please correct

Reply:
reverted

Reply:
reverted